config/initializers/gitlab.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Gitlab.configure do |config|
+  config.endpoint = GitlabConfig.api_endpoint
```

Comment:
What is the point of this change? Why passing `endpoint` to every client wasn't ok?

Reply:
- I didn't find anywhere that `::Gitlab.client` used another end point than `GITLAB_API_ENDPOINT` 
- why not reduce the number of input params if we can do it and the Gem provides such opportunity?
- on my point this
`::Gitlab.client(private_token: user_access_token)`
looks better than
`::Gitlab.client(endpoint: ENV["GITLAB_API_ENDPOINT"], private_token: user_access_token)`

Reply:
It's hardly possible, but some client might use self-hosted gitlab.
I'm not insisting on changing this back, but I guess it's just your change seemed to me not worth the effort 🙃